source/navigation/routes.tsx

```diff
@@ -299,6 +299,24 @@ const SettingsStackScreens = () => {
 				name="NetworkLogger"
 				options={settings.NetworkLoggerNavigationOptions}
 			/>
+
+			{/* Start component library */}
```

Comment:
Should there be in their own Group?

Reply:
It started out that way. I had a fully separate stack navigator written out and then I was unable to push the view from settings to the library viewer.

I'll retry it because today we can push from home to settings so I think I know what I can change to make that work. Thanks!